examples/visualization_demo/Main.cc

```diff
@@ -162,6 +162,36 @@ void buildScene(ScenePtr _scene)
   comVisual->SetInertial(comVisualInertial);
   box->AddChild(comVisual);
 
+  // create joint child visual
+  VisualPtr jointChildBox = _scene->CreateVisual("joint_child");
+  jointChildBox->AddGeometry(_scene->CreateBox());
+  jointChildBox->SetOrigin(0.0, 0.0, 0.0);
+  jointChildBox->SetLocalPosition(3.5, 0.5, 0.0);
+  jointChildBox->SetLocalRotation(0, 0, 0);
+  jointChildBox->SetMaterial(blue);
+  root->AddChild(jointChildBox);
+
+  // create joint parent visual
+  VisualPtr jointParentBox = _scene->CreateVisual("joint_parent");
+  jointParentBox->AddGeometry(_scene->CreateBox());
+  jointParentBox->SetOrigin(0.0, 0.0, 0.0);
+  jointParentBox->SetLocalPosition(2.0, 0.5, 0.0);
+  jointParentBox->SetLocalRotation(0, 0, 0);
+  jointParentBox->SetMaterial(blue);
+  jointParentBox->Material()->SetTransparency(0.5);
+  jointParentBox->Material()->SetDepthWriteEnabled(false);
+  root->AddChild(jointParentBox);
+
+  // create joint visual
+  JointVisualPtr jointVisual = _scene->CreateJointVisual();
+  jointChildBox->AddChild(jointVisual);
+  jointVisual->SetType(JointVisualType::JVT_REVOLUTE);
```

Comment:
I was a little confused at first to see 2 joint visuals while the type is `JVT_REVOLUTE`. I would expect this to be `JVT_REVOLUTE2` or `JVT_UNIVERSAL`. Maybe only create the parent axes or make it visible if joint type is revolute2 / universal?

Reply:
[`714d523`](https://github.com/ignitionrobotics/ign-rendering/pull/366/commits/714d523494950dfeed156b53082e56896ef58b59) should fix it